sql/core/src/main/scala/org/apache/spark/sql/Arrow.scala

```diff
@@ -21,81 +21,44 @@ import scala.collection.JavaConverters._
 import scala.language.implicitConversions
 
 import io.netty.buffer.ArrowBuf
-import org.apache.arrow.memory.RootAllocator
-import org.apache.arrow.vector.BitVector
+import org.apache.arrow.memory.{BaseAllocator, RootAllocator}
+import org.apache.arrow.vector._
```

Comment:
Usually explicit imports are preferred.  Is it basically an import for every vector type?